ballerina/build.gradle

```diff
@@ -1174,6 +1186,59 @@ task generateCache(dependsOn: ':cache-generator:build') {
                             ignoreExitValue true
                         }
                     }
+                    def dependencyGraphJson = file("${balaPath}/dependency-graph.json")
+                    def parsedDependencyJson = new JsonSlurper().parseText(dependencyGraphJson.text)
+                    parsedDependencyJson.modules.dependencies.each { dependency ->
+                        println(dependency.module_name.toString())
+                        println(dependency.version.toString())
+                        if ( dependency.module_name.size() > 0 ) {
+                            for ( i in 0..dependency.module_name.size()) {
+                                def moduleName = dependency.module_name[i].toString() + "-ballerina-zip"
+                                if ( sortedLibs.contains(moduleName) ) {
+                                    if ( !dependencyMap["${moduleName}"].contains(dependency.version[i].toString()))
+                                    dependencyMap["${moduleName}"].add(dependency.version[i].toString())
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        // generate caches for minor versions of Standard Libraries
+        dependencyMap.each { key, val ->
+            if (val.size > 1 && key != "observe-ballerina-zip") {
```

Comment:
This will skip modules only has 1 dependency, is it intentional?

Reply:
Yes. The first one is the version available in the pack